app/assets/javascripts/validation.js

```diff
@@ -180,11 +189,16 @@ function validatePassword(confirmPasswordElement, obj) {
 }
 
 function validateConfirmPassword(passwordElement, obj) {
-  var confirmPassword = this.value;
+  var confirmPassword = this.value.trim();
   var password = passwordElement.value;
 
-  if (confirmPassword !== password) {
-    obj.updateUI(this, false, "Passwords must be equal");
+  if (confirmPassword.length === 0) {
+    obj.updateUI(this, false, "The field can't be empty");
+    return;
+  }
+
+  if (confirmPassword !== password || !isPasswordValid(password)) {
+    obj.updateUI(this, false, "Passwords must be the same");
```

Comment:
Password and Password Confirmation should be the same.

Reply:
:+1: